Header header
string speech		#The troll will say this out loud. Write normal english.
string expression 	#shortform for the expression, like angry, smile, etc. 
float32 peak_time 	#In seconds how long it will take for the expression to reach max intensity
float32 peak_duration 	#The time in seconds the face will stay at the most intense expression
float32 fade_time 	#The time in seconds it takes to revert to a neutral expression.
